send-transaction-service/src/send_transaction_service.rs

```diff
@@ -140,53 +138,31 @@ impl Default for Config {
 pub const MAX_RETRY_SLEEP_MS: u64 = 1000;
 
 impl SendTransactionService {
-    pub fn new<T: TpuInfo + std::marker::Send + 'static>(
-        tpu_address: SocketAddr,
+    pub fn new<Client: TransactionClient + Clone + std::marker::Send + 'static>(
```

Comment:
I could have defined a type alias for that or, maybe, it is good to be explicit for these interface functionality (?).